src/main/java/mate/academy/bookshop/model/Book.java

```diff
@@ -5,31 +5,40 @@
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
+import jakarta.persistence.Table;
 import java.math.BigDecimal;
 import lombok.Data;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
 
 @Entity
 @Data
+@SQLDelete(sql = "UPDATE books SET is_deleted = true WHERE id=?")
+@Where(clause = "is_deleted = false")
```

Comment:
`@Where` is deprecated since = "6.3", so let's use `@SQLRestriction` instead
```suggestion
@SQLRestriction(value = "is_deleted = false")
```